server/v1/controllers/auth.controller.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable arrow-parens */
+/* eslint-disable consistent-return */
+/* eslint-disable comma-dangle */
+import users from '../models/user.db';
+import user from '../models/user.model';
+import validate from '../helpers/user.validation';
+import auth from '../helpers/authenticate';
+
+class userController {
+  static signUp(req, res) {
+    const { error } = validate(user(req));
+
+    if (error) {
+      return res.status(400).json({
+        status: 400,
+        message: 'Validation Error',
```

Comment:
This message is not really needed since you are displaying the validation error down.